readme.md

```diff
@@ -44,10 +44,11 @@ Heads up for Windows + Vagrant users: Start by applying the fix in [issue #61](h
 ```bash
 composer create-project jadjoubran/laravel5-angular-material-starter --prefer-dist
 cd laravel5-angular-material-starter
-#fix database credentials in .env
+#fix database credentials, add API_API_PREFIX and API_DOMAIN entries to .env
```

Comment:
- You can't add API_PREFIX & API_DOMAIN
- when you install this starter project, it copies the env.example to .env which has a defualt value for API_PREFIX
so usually you don't have to update/change these unless you want to customize it yourself. But that would break the default Restangular configuration, so I don't think it's advised..
Let me know what you think